armi/reactor/blocks.py

```diff
@@ -2332,11 +2377,41 @@ def getPinPitch(self, cold=False):
             )
 
     def getWettedPerimeter(self):
-        """Return the total wetted perimeter of the block in cm.
+        r"""Return the total wetted perimeter of the block in cm.
 
         .. impl:: Wetted perimeter of block is retrievable.
             :id: I_ARMI_BLOCK_DIMS7
             :implements: R_ARMI_BLOCK_DIMS
+
+            This implementation computes wetted perimeters for specific Components, as specified
+            by their Flags (:need:`R_ARMI_FLAG_DEFINE`). Hollow hexagons and circular pin Components
+            are supported. The latter supports both instances where the exterior is wetted 
+            (e.g., clad, wire) as well as when the interior and exterior are wetted (circular ducts).
+
+            Hollow hexagons are calculated via,
+
+            .. math:: 
+
+                \frac{6 \times \text{ip}}{\sqrt{3}},
+            
+            where :math:`\text{ip}` is the inner pitch of the hollow hexagon. Circular pin Components  
+            where the exterior is wetted is calculated via,
+
+            .. math::
+
+                N \pi \left( \text{OD}_c + \text{OD}_w \right),
+            
+            where :math:`N` is the total number of pins, :math:`\text{OD}_c` is the outer diameter
+            of the clad, and :math:`\text{OD}_w` is the outer diameter of the wire, respectively. 
+            When both the interior and exterior are wetted, the wetted perimeter is calculated as
+
+            .. math::
+            
+                \pi \left( \text{OD} + \text{ID} \right),
+            
+            where :math:`\text{OD}` and :math:`\text{ID}` are the outer and inner diameters of the pin
+            Component, respectively.
```

Comment:
In the last sentence, should it be for the hollow circle component instead of the pin component?

Reply:
Pin components are given inner and outer diameters, regardless. Solid pins have an inner diameter of zero while hollow have a non-zero inner diameter. So technically, solid and hollow are both pin components.
https://github.com/terrapower/armi/pull/1619/commits/57dcb110589a6b84e5b860818334d49bc739dc86